<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="HIBACL131408" ignored="0" name="Test"
	tests="4" failures="1" timestamp="17 Dec 2020 06:24:42 GMT" time="43.423"
	errors="0">
	<testcase name="IETest" time="26.993"
		classname="pk_Orange_HRM.Parallel_Thread_TestNG_XML_Example" />
	<testcase name="ChromeTest" time="27.494"
		classname="pk_Orange_HRM.Parallel_Thread_TestNG_XML_Example" />
	<testcase name="EdgeTest" time="38.767"
		classname="pk_Orange_HRM.Parallel_Thread_TestNG_XML_Example" />
	<testcase name="FirefoxTest" time="43.336"
		classname="pk_Orange_HRM.Parallel_Thread_TestNG_XML_Example">
		<failure type="org.openqa.selenium.remote.UnreachableBrowserException"
			message="Error communicating with the remote browser. It may have died.
Build info: version: &amp;apos;4.0.0-alpha-3&amp;apos;, revision: &amp;apos;8c567de6dc&amp;apos;
System info: host: &amp;apos;HIBACL131408&amp;apos;, ip: &amp;apos;10.45.94.243&amp;apos;, os.name: &amp;apos;Windows 10&amp;apos;, os.arch: &amp;apos;amd64&amp;apos;, os.version: &amp;apos;10.0&amp;apos;, java.version: &amp;apos;1.8.0_231&amp;apos;
Driver info: driver.version: ChromiumDriver
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 87.0.664.60, javascriptEnabled: true, ms:edgeOptions: {debuggerAddress: localhost:54610}, msedge: {msedgedriverVersion: 87.0.664.60 (23fbc5c433ca16..., userDataDir: C:\Users\adixit\AppData\Loc...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '4.0.0-alpha-3', revision: '8c567de6dc'
System info: host: 'HIBACL131408', ip: '10.45.94.243', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_231'
Driver info: driver.version: ChromiumDriver
Capabilities {acceptInsecureCerts: false, browserName: msedge, browserVersion: 87.0.664.60, javascriptEnabled: true, ms:edgeOptions: {debuggerAddress: localhost:54610}, msedge: {msedgedriverVersion: 87.0.664.60 (23fbc5c433ca16..., userDataDir: C:\Users\adixit\AppData\Loc...}, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:virtualAuthenticators: true}
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:597)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:633)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:637)
	at org.openqa.selenium.remote.RemoteWebDriver.quit(RemoteWebDriver.java:473)
	at org.openqa.selenium.chromium.ChromiumDriver.quit(ChromiumDriver.java:193)
	at pk_Orange_HRM.Parallel_Thread_TestNG_XML_Example.FirefoxTest(Parallel_Thread_TestNG_XML_Example.java:24)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.io.UncheckedIOException: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:44116
	at org.openqa.selenium.remote.http.okhttp.OkHandler.makeCall(OkHandler.java:63)
	at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.okhttp.OkHandler.execute(OkHandler.java:52)
	at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
	at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:55)
	at org.openqa.selenium.remote.http.okhttp.OkHttpClient.execute(OkHttpClient.java:45)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:576)
	... 18 more
Caused by: java.net.ConnectException: Failed to connect to localhost/0:0:0:0:0:0:0:1:44116
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.kt:270)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.kt:176)
	at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.kt:236)
	at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.kt:109)
	at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.kt:77)
	at okhttp3.internal.connection.Transmitter.newExchange$okhttp(Transmitter.kt:162)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.kt:35)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.kt:82)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.kt:84)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.kt:71)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:112)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.kt:87)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.kt:184)
	at okhttp3.RealCall.execute(RealCall.kt:66)
	at org.openqa.selenium.remote.http.okhttp.OkHandler.makeCall(OkHandler.java:60)
	... 27 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.waitForConnect(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at okhttp3.internal.platform.Platform.connectSocket(Platform.kt:126)
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.kt:268)
	... 46 more
... Removed 10 stack frames]]>
		</failure>
	</testcase> <!-- FirefoxTest -->
</testsuite> <!-- Test -->
